contents/blog/data-management-feature.md

```diff
@@ -0,0 +1,92 @@
+
+And when you click 'Edit' you can add a description, add tags or mark it as a Verified Event.
+
+![PostHog - write definition hover card](../images/blog/data-management-feature/write_definition_card.png)
+
+Alternatively, click on 'View' in the definition hover card and you'll be redirected to the new Data Management page, where you can explore all the Events and Event Properties you've created.
+
+[WIP screenshot]
+
+And if you want a gold star in best practice, you can also opt to create definitions with our API. Check out our documentation on how to use the [`event_definitions`](/docs/api/event-definitions) and [`property_definitions`](/docs/api/property-definitions) API endpoints.
+
+Oh, and in case you were wondering, the awesome thing about definitions in PostHog is that they can be created, updated, or deleted at any time without affecting the event data that is already there, so **you don't need to create event definitions before ingesting events**. 
+
+## How to use Data Management as a team (best practices)
+
+To help you in your journey to data management nirvana, here are a few rules of thumb to consider when using these new features.
+
+1. The more definitions you add, the easier it will be for your teammates to understand what each event and property represents.
+
+Take a custom event sent as `sign_up` for example that is triggered on user sign up. Pretty self explanatory, right? Imagine you have a second app pushing data to the same PostHog instance sending sign up events as `sign up`. Or maybe you send a `sign up` event from your frontend and one from your backend. A new teammate using PostHog will find it impossible to differentiate between the two, potentially leading to incorrect conclusions and frustration. Describing these events properly can ensure your teammates find exactly what they're looking for.
+
+2. Official organization-wide events should be verified so that teammates can discover and use them easily. 
+
+Verified events will be prioritized in filters and other selection components to signal to the rest of the team that this event should be used in favor of other similar events.
+
+3. Tagging is an easy way to categorize events into custom groups, be it for a product, feature or new page. 
+
+We highly recommend using tags to organize events coming in from different parts of your product. 
+
+## What's next for Data Management?
+
+To recap, the new Data Management experiences means you can:
+
+- Organize your definitions with tags
+- Manage event definitions on the fly while building insights
+- Verify trusted definitions which are prioritized in the UI
+- Easily manage all your Events, Event Properties and Actions in the Data Management tab
+- Debug your incoming events using in the Live Events tab
+
+These are powerful tools for organizing your data, but we're just scratching the surface. [AV: Let's elaborate a little more here on the roadmap. Only needs to be top-level]
```

Comment:
We need to a few little details about what's on our roadmap; get people excited about what's coming soon.